src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalCompletionService.ts

```diff
@@ -551,7 +551,8 @@ export class TerminalCompletionService extends Disposable implements ITerminalCo
 }
 
 function getFriendlyPath(uri: URI, pathSeparator: string, kind: TerminalCompletionItemKind, shellType?: TerminalShellType): string {
```

Comment:
Also we should make have some tests around this function, not sure if they exists but we should definitely have a unit test for this particular case.